6.5.group_by.py

```diff
@@ -0,0 +1,8 @@
+def group_by(func, iterable):
+    mydict = {}
```

Comment:
if you want to use 'mydict' rename to 'my_dict' but the best name here would probably be 'dictionary'

Reply:
fixed it